### PR TITLE
[Feature] Theme toggle

### LINKED ISSUE #43: [FEATURE] Theme toggle
### Description

- Any modern web app these days support a light and dark themes, this website is no different
- We need a toggle switch to toggle between light and dark themes using CSS variables.
- Initially the value should be system default theme.
- Then the changed theme value must be stored in local storage.

### Screenshots

download

### Additional information

- Always use styled components instead of plain CSS and MUI components.
## Happy contributing. 💐
Star my other Repositories [here](https://github.com/warmachine028)

### PR DESCRIPTION
**Pull request by avinasdube**
# Issue #43 📐

- I [Avinash Dubey](https://github.com/avinasdube) have worked for #43

## Guidelines 🔐

I accept the fact that I have followed the guidelines and have not copied the codes from around the internet

- [x] Contribution Guidelines
- [x] Code of Conduct

## Issue to be closed 🛅

- My pull request closes #43 

## Screenshots 📷

Here are the pictures of changes that I have made 🔽

In Light Mode :
  
LightMode

In Dark Mode :

---
DarkMode

### CODE REVIEW

File: server/.env.example
Comment (warmachine028) [inline server/.env.example:1]: This was an just an example file to set up `.env.local`, its contents are for demonstration only. It needs to be reverted back to original.

**Review by warmachine028 — CHANGES_REQUESTED** [review]

**Comment by avinasdube** [comment]
I have fixed all the suggestion you requested. I hope everything is up, running and good now.

**Comment by warmachine028** [comment]
Great work! I would be reviewing this PR shortly.

**Comment by avinasdube** [comment]
> Great work! I would be reviewing this PR shortly.

Hey, What happened ? Still not reviewed !

**Review by warmachine028 — APPROVED** [review]

**Comment by avinasdube** [comment]
Thank You for giving me opportunity to make my little contribution in your project. 🙏🏻

**Comment by warmachine028** [comment]
Keep progressing like this 🚀. Wishing you for more such moments !

**Pull request merged**